sdks/nodejs/src/index.js

```diff
@@ -13,5 +13,7 @@
 // limitations under the License.
 
 const AgonesSDK = require('./agonesSDK');
+const AlphaAgonesSDK = require('./alphaAgonesSDK');
 
-module.exports = AgonesSDK;
+module.exports.sdk = AgonesSDK;
```

Comment:
This is a breaking change how the SDK would have to be imported:
`const AgonesSDK = require('@google-cloud/agones-sdk');`
&rarr;
`const AgonesSDK = require('@google-cloud/agones-sdk').sdk;`

Also I now realise the case is inconsistent.

Reply:
Do we have to have this breaking change? Is there any way around it?

Reply:
Sure, I have a couple of ideas